tests/python_package_test/test_basic.py

```diff
@@ -328,3 +328,42 @@ def check_asserts(data):
         lgb_data.set_weight(sequence)
         lgb_data.set_init_score(sequence)
         check_asserts(lgb_data)
+
+    def test_trees_to_dataframe(self):
+
+        def _imptcs_to_numpy(X, impcts_dict):
+            cols = ['Column_' + str(i) for i in range(X.shape[1])]
+            imptcs = []
+            for col in cols:
+                try:
+                    imptcs.append(impcts_dict[col])
+                except KeyError:
+                    imptcs.append(0.)
```

Comment:
I think it can be replaced with one-line solution `imptcs.append(impcts_dict.get(col, 0.))`

Reply:
Changed in the final (6th) commit.